.github/workflows/deploy_tre_reusable.yml

```diff
@@ -287,6 +287,7 @@ jobs:
           TF_VAR_api_client_secret: "${{ secrets.API_CLIENT_SECRET }}"
           TF_VAR_stateful_resources_locked:
             "${{ github.ref == 'refs/heads/main' && inputs.prRef == '' && true || false }}"
+          TF_VAR_enable_airlock_malware_scanning:  "${{ secrets.ENABLE_AIRLOCK_MALWARE_SCANNING }}"
```

Comment:
This is an example of where this var is not used in the underlining make command. Perhaps you should just pass it where it's actually needed (in the deploy-core target)